bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx68.html

```diff
@@ -0,0 +1,85 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/whatsnew_67" %}
+
+{% extends "firefox/whatsnew/whatsnew-fx67.html" %}
+
+{% block fxa_form %}
+  {{ fxa_email_form(
+      entrypoint='mozilla.org-whatsnew67',
+      button_class='mzp-c-button mzp-t-product mzp-t-small',
+      utm_source='whatsnew',
+      utm_params={ 'campaign': 'wnp68', 'content': '/firefox/WNP/68/', 'source': 'wnp-68'})
+  }}
+{% endblock %}
+
+{% block benefits %}
+<div class="mzp-l-content c-section-benifits">
+
+    {# L10n: "Monitor" is a proper name; it should be capitalized and should not be translated. #}
+    {% call feature_card(
+      title=_('Keep an eye on data breaches with Monitor'),
+      ga_title='Check Out Monitor',
+      link_cta=_('Check Out Monitor'),
+      link_url='https://monitor.firefox.com/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=wnp68&utm_content=try-send&entrypoint=mozilla.org-whatsnew68',
+      image_url='firefox/whatsnew_67/monitor.svg',
+      include_highres_image=False,
+      aspect_ratio='mzp-l-card-feature-left-66',
+      class='wn67-lockwise'
```

Comment:
maybe `wnp68-lockwise` :)

Reply:
I don’t think there’s any reason to change the CSS class names, unless i’m missing something as to why there are named this way?

Reply:
You are absolutely right! I was on autopilot for 67's. Feel free to ignore. 🤦‍♀